Corgstation 4

PowerfulBacon opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

The 4th iteration of a map I've been casually doing whenever I don't feel like coding is finally complete.
Corgstation is an entirely new and completely original station with a unique design (Whether or not this is a good thing is still to be found).
The map features absolutely beautiful lighting and introduces a new glowsticks spawner that will auto light glowsticks on the groups for giving a colourful and pretty glow to maints.
No one really cares about the text, let's get to the pretty pictures.

I'm quite happy how most of it turned out, brig physician is going to have a good time, although I think atmospherics is a little off.

The engine used is a standard supermatter with valves that can be opened and closed depending on whether or not you want to connect it to the TEG, which can also be connected to the cold loop or turbine. (It's pretty customisable, you can choose what you want connected by opening and closing valves.)
Robotics is extremely nice, very happy with how that turned out.
Cargo I think is quite nice too and I like security.
Genetics and medbay storage are pretty good.
Evac is beautiful and cozy.

Also the maintenance game is very strong for you assistants out there.

![image](https://user-images.githubusercontent.com/26465327/101214262-a1e1c980-3673-11eb-996d-2fc08374a369.png)

Also the prisons are super nice for your long term stays.

## Composite Map

(OUTDATED DUE TO UPDATES, SEE CHECKS FOR UP TO DATE IMAGE)

<a href="https://ibb.co/cJHnd5b"><img src="https://i.ibb.co/cJHnd5b/corgstation-full.png" alt="corgstation-full" border="0"></a>

![image](https://user-images.githubusercontent.com/26465327/101214420-ea00ec00-3673-11eb-8e37-5c7901fc384c.png)

## Highlights

My hand-selected favourite parts of the map.

![image](https://user-images.githubusercontent.com/26465327/101214479-000eac80-3674-11eb-8cd2-aa7d99797cae.png)

![image](https://user-images.githubusercontent.com/26465327/101214508-0bfa6e80-3674-11eb-9809-6fc1d0d37ddf.png)

![image](https://user-images.githubusercontent.com/26465327/101214528-1288e600-3674-11eb-981e-2eb7e4e4f706.png)

![image](https://user-images.githubusercontent.com/26465327/101214563-203e6b80-3674-11eb-8f1c-a00afa7f337f.png)

![image](https://user-images.githubusercontent.com/26465327/101214618-351aff00-3674-11eb-8557-2791a797c3bd.png)

![image](https://user-images.githubusercontent.com/26465327/101214634-3a784980-3674-11eb-94c7-c888ea320924.png)

![image](https://user-images.githubusercontent.com/26465327/101214653-419f5780-3674-11eb-9b40-cadb990ea347.png)

![image](https://user-images.githubusercontent.com/26465327/101214684-4cf28300-3674-11eb-8bbe-13fc96d6a555.png)

![image](https://user-images.githubusercontent.com/26465327/101214703-54199100-3674-11eb-85da-b876971608a8.png)

![image](https://user-images.githubusercontent.com/26465327/101214726-5e3b8f80-3674-11eb-85c5-72f03044d40f.png)

![image](https://user-images.githubusercontent.com/26465327/101214742-65629d80-3674-11eb-8822-f86745422488.png)

![image](https://user-images.githubusercontent.com/26465327/101214755-6b587e80-3674-11eb-9f6a-5e27e677151e.png)


## Why It's Good For The Game

A pretty nice new map.

## Changelog
:cl:
add: Corgstation
add: Lit glowsticks spawner.
add: Corgstation code ownership
add: Corgstation Emergency Shuttle
add: Corgstation Arrivals Shuttle
fix: Fixes force_map not actually working
tweak: Allows shuttles to be created in area/science/shuttle
tweak: Pins some important vars on fastdmm2 for obj/machinery/camera
tweak: Allows anchored supermatter crystals to be launched via those chapel cannon thingys
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

PowerfulBacon committed:
corgstation1

PowerfulBacon committed:
<3

PowerfulBacon committed:
Science

PowerfulBacon committed:
based

PowerfulBacon committed:
coeh

PowerfulBacon committed:
a

PowerfulBacon committed:
e

PowerfulBacon committed:
based

PowerfulBacon committed:
cargo

PowerfulBacon committed:
based

PowerfulBacon committed:
Security + Engineering + Shuttle Construction + Atmos

PowerfulBacon committed:
epic

PowerfulBacon committed:
Corgstation <3

PowerfulBacon committed:
corg<3

PowerfulBacon committed:
Merge branch 'master' into corgstation_4

Commit:
Update CorgStation.dmm

Commit:
Updates to corgstation

Commit:
Fixes the jason

Commit:
Update _basemap.dm

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
Virology, atmos a bit

Commit:
Update CorgStation.dmm

Commit:
Moves forcemap to where it actually fucking works

Commit:
Science Shuttle Construction, glowsticks

Commit:
Corgstation Shuttle

Commit:
Update emergency_corg.dmm

Commit:
vaa

Commit:
Update CorgStation.dmm

Commit:
Atmospherics

Commit:
Merge remote-tracking branch 'upstream/master' into corgstation_4

Commit:
abomination to god

Commit:
a bunch of shit

Commit:
Update CorgStation.dmm

Commit:
Intercoms, Request Consoles, Light switches, APCs, Air Alarms

Commit:
Corgstation

Commit:
Update CorgStation.dmm

Commit:
HOS office

Commit:
Finish

Commit:
Bar Doors

Crossedfall commented:
ooooooooo

PowerfulBacon commented:
ah shit I failed the tests 😔

Commit:
Corgstation Aux dock

Commit:
Comments out forcemap

PowerfulBacon commented on .github/CODEOWNERS:
```diff
@@ -28,6 +28,7 @@ TGS3.json @crossedfall
 
 /code/game/machinery/shuttle @powerfulbacon
 /code/modules/shuttle @powerfulbacon
+_maps/map_files/CorgStation @powerfulbacon
```
```suggestion
/_maps/map_files/CorgStation @powerfulbacon
```

PowerfulBacon committed:
Update .github/CODEOWNERS

PowerfulBacon commented on code/datums/map_config.dm:
```diff
@@ -54,7 +54,7 @@
 		return
 
-	json = rustg_file_read(json)
+	json = file2text(json)
```
```suggestion
	json = rustg_file_read(json)
```

PowerfulBacon committed:
Update code/datums/map_config.dm

ghost commented:
What happened to Corgstation 1, 2 and 3?

Commit:
Fills some empty space in chapel

Commit:
Removes dirty variables

Commit:
Merge branch 'corgstation_4' of https://github.com/PowerfulBacon/BeeStation-Hornet-Bacons-Fork into corgstation_4

PowerfulBacon commented:
> 
> 
> What happened to Corgstation 1, 2 and 3?

1 I made entirely without saving (actually I'm pretty sure I saved it a few times and backed it up to git but I still managed to lose it), 2 and 3 were shit and I restarted early on.

Commit:
Fixes the stupid air alarm being stupid

Commit:
Fixes undefined contraband type

Commit:
Adds a few beacons around and spices up HOP's office

PowerfulBacon commented:
Horray it passes

PowerfulBacon commented:
oh fuck I forgot the labour shuttle shit

ike709 commented:
Did you include all of the miscellaneous spawners in space/maint and did you proper use the nearstation area? Those are usually the biggest gotchas for wholly original maps.

ike709 commented:
Map looks pretty good though.

Commit:
Labour (Labor?) Shuttle

PowerfulBacon commented:
> 
> 
> Did you include all of the miscellaneous spawners in space/maint and did you proper use the nearstation area? Those are usually the biggest gotchas for wholly original maps.

Station uses nearstation, and yea I forgot the carp spawners for most of the station

ghost commented:
![image](https://user-images.githubusercontent.com/47011211/101221758-a6fd4380-3688-11eb-968d-619e004ef01f.png)
wtf bacon, why can you stare at people showering?

LastCrusader105 commented:
EndoStation best station

legit doe wheres the full station image?

St0rmC4st3r commented:
Is it intended to be a lowpop map?

Crossedfall commented:
> 
> 
> EndoStation best station
> 
> legit doe wheres the full station image?

You get them from the checks.

https://github.com/BeeStation/BeeStation-Hornet/pull/3085/checks?check_run_id=1501545349

park66665 reviewed (dismissed):
![Screenshot 2020-12-05 140740](https://user-images.githubusercontent.com/8010007/101234601-2da33680-3704-11eb-8f4b-781faf792cbe.png)

Found at X 158 Y 148. It shouldn't be a manifold. Weird pipe connections are not just a map bug; preliminary results show that such mis-connections might sabotage pipeline code functions occasionally.

This map is great anyway, don't worry much about it. Box probably has multiple of them and it doesn't break BYOND.

LastCrusader105 commented:
> This map is great anyway, don't worry much about it. Box probably has multiple of them and it doesn't break BYOND.

Box won’t when they merge my PR, together we will bring a new standard to mapping!

PowerfulBacon commented:
> ![image](https://user-images.githubusercontent.com/47011211/101221758-a6fd4380-3688-11eb-968d-619e004ef01f.png)
> wtf bacon, why can you stare at people showering?

There are shutters haha

PowerfulBacon commented:
> EndoStation best station
> 
> legit doe wheres the full station image?

Look at composite image in the pr description for a full render of the map with lighting

PowerfulBacon commented:
Needs a delivery beacon network, that pipe fixed and generic event spawned.

Commit:
Event spawns, fixed pipes

Commit:
Nav beacons

PowerfulBacon commented:
Ready

Commit:
Deblanding of science

Commit:
A few signs and a little more flavour

Commit:
Disposals Ejection Room

Commit:
Jazzes up the space around the station

Commit:
Update CorgStation.dmm

Commit:
Corgstation cargo shuttle

Commit:
Fixes cargo ship

ZombiesVsPlants commented:
Looking at the Ai sat as an ai main I can see 4 problems so far
1. Where are those mounted flashes? Without them, there is nothing stopping an unsynced borg from just waltzing in and bludgeoning your core to death since the "Target cyborgs" option for the turrets doesn't actually work.
2. Where are the shutters? On Metastation you have 2 buttons to put shutters
Here:
![image](https://user-images.githubusercontent.com/60122079/101272076-ef6d4c00-37c3-11eb-9a57-574403968ef9.png)
and the next best place I'd assume here:
![image](https://user-images.githubusercontent.com/60122079/101272097-04e27600-37c4-11eb-8d71-b1f78f2d79a9.png)
On the windows and door
The first shutter is required otherwise people can just break in through the top and shoot you with a laser and kill you outright.
The 2nd shutter is not AS necessary but it was a nice layer of defense on Metastation and would be required to defend your apc if you added point 3.
3. The APC at the back, while very defensible, only encourages people to break in through the top and block using unsecured girders or closets, and breaking the apc is an instant win. I would move it to the front to encourage people to come through the front door and force people going in the backway to fight the ai core head-on. (Something the ai can defend against with pingsky, flashes, turrets, and soap)
4. I can see VERY few cameras in the ai sat, 1 camera in the core I doubt would cover the entire core room and nothing is sadder than having no vision in your very own core room.
Yeah comparing this and metastation I can only see 1 camera in the carpstation core and 4 cameras in the metastatation core (Which is far smaller)

I would also worry if the single SMES is enough to power the many more turrets, Telecomms, and ai upload for 2 hours, though beestation might have far faster rounds than AUstation.

Other than that it looks perfect and I'm looking forward to playing it.

ZombiesVsPlants commented:
OH also 
![image](https://user-images.githubusercontent.com/60122079/101272330-3a885e80-37c6-11eb-921a-bc24ec0fb40f.png)
how is the captain meant to turn off the turrets before entering?

ZombiesVsPlants commented:
also, I would appreciate a few more holopads for the ai core if at all possible.

ZombiesVsPlants commented:
oh and no cyborg recharger as far as I can see.

PowerfulBacon commented:
> 
> 
> OH also
> ![image](https://user-images.githubusercontent.com/60122079/101272330-3a885e80-37c6-11eb-921a-bc24ec0fb40f.png)
> how is the captain meant to turn off the turrets before entering?

The turret controller for that room is just before the bridge thing

PowerfulBacon commented:
Valid points, I plan to rework medical and AI core today

PowerfulBacon commented:
![Uploading image.png…]()

PowerfulBacon commented:
![image](https://user-images.githubusercontent.com/26465327/101278028-73b5d400-37b0-11eb-8eb6-ecf5a34a3dcd.png)

Commit:
AI Core Changes

Commit:
Updates the AI core

Commit:
Virology

Commit:
2 solar panels

Commit:
vault

Commit:
Removes floating thing

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
area cleanup

Commit:
Authentication panels

PowerfulBacon commented:
After some work changes have been made:

Medical:
 - Old cryo room replaced with morgue.
 - Cryo pods moved next to sleeper room.
 - Chemistry has been reorganised and a chem wardrobe added.
 - Added shutters to CMO's office.

Permabrig:
 - Added in exterior protection.

AI Core:
 - Relocated turrets
 - Added shutters to windows
 - Added mounted flashers
 - Added cyborg charging station

Robotics:
 - Added cyborg charging station

Chapel:
 - Removed maintenance room
 - Added crematorium

Head of Personnel's office:
 - Reduced in size and extra space added to maints
 - Added 2 posters

Security: Added the prisoner transfer center

Virology:
 - Added space reinforcements
 - Linked disposals up
 - Added 2 space suits

Also:
 - Added solars

Commit:
adds a fan next to a xeno spawn

Commit:
Adds medbay to the rainbow sign

Commit:
adds some space detail

Commit:
adds 1 office chair

Commit:
Fixes the variables for the corgstation labour shuttle being reset

Commit:
Adds a tiny fan to corg cargo shuttle

Commit:
Fixes education chamber

Commit:
Update labour_corg.dmm

Commit:
Fixes a wrong area type

Commit:
Adds some shit to teleporter room

Commit:
Fixes up some decals

Commit:
Adds 1 turf_decal

Commit:
Links up wires

Commit:
Fixes telecomms monitoring

Commit:
Remove redundent code

ZombiesVsPlants commented:
A camera next to the primary core would be greatly appreciated.

Commit:
Makes science a bit more fancy

Penwin0 commented:
I am unable to find any major gripes with the map, its good

ghost commented:
I don't like how there are two spare cartons of Synthflesh in genetics, cloning should be discouraged not encouraged.

PowerfulBacon commented:
> 
> 
> I don't like how there are two spare cartons of Synthflesh in genetics, cloning should be discouraged not encouraged.

Cloning and surgery should be encouraged and used for their intention: Surgery for reviving simple damages / dead bodies, cloning for the harder to heal stuff like husks or brains. Pretty sure all other maps have synthflesh around

ghost commented:
> Pretty sure all other maps have synthflesh around

No, they don't, all other maps only have one Synthflesh carton in the CMO's locker, that's it, and it's for a good reason.

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
escape pods

Commit:
Update CorgStation.dmm

PowerfulBacon commented:
Added:
 - Gateway
 - Escape Pods

PowerfulBacon commented on interface/interface.dm:
```diff
@@ -264,6 +264,7 @@ Any-Mode: (hotkey doesn't need to be on)
 		if("MetaStation")			map_in_url = "meta"
 		if("Kilo Station")          map_in_url = "kilo"
 		if("PubbyStation")          map_in_url = "pubby"
+		if("CorgStation")			map_in_url = "corg"
```
Will need website update

PowerfulBacon commented:
There is no PDA server

PowerfulBacon commented:
Solars need solars

PowerfulBacon commented:
no sugar or flower

PowerfulBacon commented:
No genetics computer in botany

PowerfulBacon commented:
No science servers

PowerfulBacon commented:
Botany doesnt get their disk stuff, no grinder. no tables. No kitchen access. no chem machine

PowerfulBacon commented:
toxins chamber doors dont open

Commit:
Adds panels to the solar

Commit:
Anchors the 'anchored' supermatter crystal shard

Commit:
Plugs in the TEG

Commit:
Fixed library maints, added research servers, fixed a door leading into a grille

Commit:
Gives kitchen resources and fixes the belt

Commit:
Adds a door to kitchen

Commit:
Fixes botany

Commit:
Fixes gateway access, fixes medbay deliveries

Commit:
security checkpoint access fixes

Commit:
Fixes robotics access

Commit:
Improves medbay security

Commit:
Removes a medvendor from medbay

Commit:
Adds more blood to medbay to cover yourself with

Commit:
Adds a windoor to the ORM

Commit:
Improves HOS' office security

Commit:
Adds more oxygen to toxins

Commit:
Adds in HOP line buttons

Commit:
Updates the corg sign

Commit:
adds some random plants and closets

Commit:
Fixes TEG pipe direction

Commit:
N2 canisters

Commit:
Adds spawns for secmains

Commit:
Xenobio monkey cubes

Commit:
Adds lattices to engineering

Commit:
Removes corgstation nightlighting

PowerfulBacon commented:
shaft miners, have no cargo access. and therefore cant go to mining station

PowerfulBacon commented:
one of the xenobio slime cells is missing walls and its just spaced

PowerfulBacon commented:
I dont think sec starts with a techfab still

PowerfulBacon commented:
Antique gun

Commit:
Adds in 2 traitor objectives

Commit:
Security techfab

Commit:
Reinforces command hall doors

PowerfulBacon commented:
no tracking implants

Commit:
Improves security around armoury and vault a bit

Commit:
adds in perma brig things to perma brig

PowerfulBacon commented:
north-east maint still lacks an airlock

PowerfulBacon commented:
i cant open the sec checkpoint near the shuttle as a secoff

Commit:
water tank outside janies room

Commit:
Adds labour monitors

Commit:
Adds more science spawns

Commit:
Cleans up northeast maints

Commit:
Better TEG pipes

Commit:
Better maint doors

Commit:
improved maint doors 2

Commit:
Update CorgStation.dmm

Commit:
adds in a projector to prevent beams going into atmos

Commit:
Update CorgStation.dmm

Commit:
fixed cold gateway

Commit:
Adde a cyclelink to telecomms

Commit:
removes shotguns from sec

Commit:
Update CorgStation.dmm

Commit:
Maintifies the maints

Commit:
Update CorgStation.dmm

Commit:
Engineering airlock

PowerfulBacon commented:
Reee fastdmm is really fucking annoying and keeps resettings variables of buttons for no reason

Commit:
Fixes some buttons

Commit:
Update CorgStation.dmm

Commit:
adds a light

Commit:
pre science rework

Commit:
dorms maints rad protection

Commit:
Adds Xenobiology BZ

Commit:
The great extuinguisher update

Commit:
Moves the chaplain's drobe

Commit:
Adds a new rock satellite for a connection between main maints and sec/cargo maints

Commit:
Centers the entire map by moving it over

Commit:
Complete Science Overhaul

Commit:
Hides some pipes

Commit:
Puts AI sat cams on a seperate network

PowerfulBacon commented:
add coffee machine

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
Improves requests consoles

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
r

Commit:
zxcv

Commit:
Update CorgStation.dmm

Effemu commented:
okay so 2 things

- med needs prosthetics box
- surgery room doesnt have circular saw and retractor

there was something else about med too but i forgot what it was

PowerfulBacon commented:
assistant has access to clown's room

Sinestia commented:
cat ears in bomb test site :'(

doofus9341 commented:
- security camera in engineering by the emitters should be moved since the emitters destroy it when you turn them on
- some rapid cable layers in engineering would be nice

Commit:
Update CorgStation.dmm

PowerfulBacon commented:
ill do some more testmerging soon (tm)

PowerfulBacon commented:
Needs more holopads

KennyBadrige commented:
hey, after playing cargo for a bit, i compiled my gripes into an imgur album: https://imgur.com/a/Bn8TqfR

Commit:
Update CorgStation.dmm

Cenrus commented:
Dorms doesn't have a ClothesMate vendor, unless I am blind

Cenrus commented:
Nightmare was unable to spawn on the map, does it have xeno spawn points?

PowerfulBacon commented:
> 
> 
> Nightmare was unable to spawn on the map, does it have xeno spawn points?

Yes, many.
Note sure whats up with nightmare they have been failing spawn checks on maps like meta recently

Commit:
makes ians gf only 10% chance

Commit:
New emergency shuttle

Commit:
Fixes emergency shuttle

PowerfulBacon commented:
redundant buttons in the old checkpoint

PowerfulBacon commented:
No console to pay off your fines

PowerfulBacon commented:
bridge grilles need shocking

PowerfulBacon commented:
no virus food

4rchibald commented:
I just had my first experience with a map and I enjoyed it.
I really like the fact that we're finally breaking from Box and Meta designs and as engineer I especially enjoyed engineering department. Whole map has a modern vibe with conveyor belts used to transport food from kitchen too bar and purposefully designed room for shuttle construction and plumbing. 
Is that map inspired by something? I had a feeling of deja vu when I saw location of gateway, it took me back to my first days in SS13

Smaller issues I noticed:
-It seems that SM engine shutters would block emitter beam when lowered so I would remove middle shutter
-CE office could use some more windows since you can't peak inside it at all from engineering foyer
-Atmos desk seems to be a missing feature 
-Something that bothered me while checking map as ghost...shouldn't that curved railings at chemistry be placed the opposite way?

Bigger issues and suggestions that probably won't be ever changed since they are already too far gone and incorporated deeply into stations layout:
-Wasn't there a policy that we want distro and waste pipes on the same layers? Either I remember it completely wrong or this map goes against that design. I'm pretty sure Box was getting a lot of heat for its pipe layers and APC in maints
-As one of the few people who actually play with tcomms (fix them as engineer and mess with them as traitor) I am completely against putting them on AI sat. That really makes them closely guarded by AI who bolts them down and frequently check for its own safety. That makes it too hard and annoying to both fix and sabotage so I would suggest moving them inside the station
-Same goes for upload and vault
-I was working on medbay revamp for Box few months ago before I got a break from SS13. Of all the changes made I actually liked connection of chemlab, plumbing room and med storage. Felt it really worked well for chemistry-medical cooperation since it all connects at one spot. As doctor you don't need to go to storage to check for meds first and then look for chemist if meds are not there, chemists have easy way of delivering meds as well. Maybe if you will be messing with medbay layout at any time it would be worth trying fitting something similar in

![95491561-0d702900-099a-11eb-8245-dca5149009c1](https://user-images.githubusercontent.com/54917446/107157589-2fe47400-6985-11eb-85e5-6670ea20b3ac.png)
-

heepox commented:
sec/cargo maint is a bit too isolated from main hall.

PowerfulBacon commented:
> -Wasn't there a policy that we want distro and waste pipes on the same layers? Either I remember it completely wrong or this map goes against that design. I'm pretty sure Box was getting a lot of heat for its pipe layers and APC in maints

Pretty sure the policy was to have distro on layer 1 and the scrubbers on layer 3 and some older maps were updated to have this, although some maps use layer 2 for distro instead of later 1.

PowerfulBacon committed:
Merge branch 'master' into corgstation_4

Commit:
Update CorgStation.dmm

Commit:
Bar rework

Commit:
Updates the bar

Commit:
Adds in pods to the sat

Commit:
Communications Rework

Commit:
Fixes emitters hitting the rad shutters

Commit:
Holopad

Commit:
Virus food, brig electrification, pepper spray

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
updates shuttle

Commit:
Minor updates

MNarath1 commented:
Atmos needs radsuits and plasma tanks in the tank storage no real reason why there should be non in them and radsuit is really needed especially for fusion/turbin also is it intentional that there is no access for atmos tech to the TEG?

MNarath1 commented:
And the scrubbers inside the turbine should have their own area/air alarm to prevent contamination thanks to someone setting air alarm in atmos to contaminated

KennyBadrige commented:
SEC corgstation feedback
-no sec outposts anywhere
-brig literally has no sec consoles
-officers get the short end of the stick in terms of rooms, gear room thats it
-gulag shuttle takes up way too much space in relation to its utility
-gulag prisoners after collecting points come out infront of perma and the hos office and need to be escorted out of the brig. Otherwise leaving them with their items to break open perma.
-brig medical is way too huge.
-armory is only accesible via warden or hos bolting open 3 doors and a 1x1 corridor.
-airlocks right side of perma dont have firealarm doors.
-whole SE of the station serves no purpose but maint loot.

Honestly, i think it would be preferable to completely re-do the brig, in its current state, its infuriating to play on.

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
Atmospheric changes

Commit:
Security rework

Commit:
Improves tcomms APC, restricts entry to the shuttle zone

Commit:
Fixes broke brig phys access, upgrades tcomms power

Commit:
Robotics power cells

Commit:
brig escape airlocks

Commit:
implanters

Commit:
Update CorgStation.dmm

Commit:
Adds sink to kitchen

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
general fixes

Commit:
adds crayons

Commit:
adds more spawns

FortSumter commented:
Did you add a dedicated custom shuttle construction yard? Would be nice if the custom station you built would give players the option to experience your other major contribution... :)

PowerfulBacon commented:
> 
> 
> Did you add a dedicated custom shuttle construction yard? Would be nice if the custom station you built would give players the option to experience your other major contribution... :)

yes

Sinestia commented:
:(

PowerfulBacon committed:
Merge branch 'master' into corgstation_4

Commit:
Atmospherics Rework

Commit:
Update CorgStation.dmm

Commit:
Fixes science thing, adds altar of the gods, fixes cargo shuttle missing fans

Commit:
Fixes weird atmos pipes

Commit:
Weakens bridge hallways access, moves the disposal unit in the toxins site

4rchibald commented:
@PowerfulBacon 

I'm really pleased to see Corg getting closer and closer to its final, finished state.
I'm also grateful that some of my suggestions were taken under consideration and you moved tcomms to station (I honestly doubt anyone else shared any kind of opinion about tcomms), that's a much better location for both crew and traitors to access them and I really like how they look.

Don't think anyone actually expressed appreciation for request consoles in almost every room, I hope it will make people use them more often.


I didn't have time to really participate in round that Corg was on but still found few small issues

First of all, why was conveyor system in kitchen removed? I actually really liked it. Corg station finally breaks away from standards set by Box and Meta, its not square but has all sorts of curves and bends , its not grey but colourful and its not static but interactive. Conveyor system to deliver food was actually a pleasing solution that both gave station some motion and wasn't some scripted gimmick, instead was something that players could replicate from tools and objects they are given, which I'm a big fan of. I'm curious about reasoning, was that players feedback or it didn't work as you intended?
That said, if you are really set on removing it you forgot about lever

![lever](https://user-images.githubusercontent.com/54917446/114089791-dd590280-98b6-11eb-834f-15787a302262.png)

When I was space walking to wire solars I was surprised by turrets shooting at me, there should be some kind of barrier to either block turrets line of sight or simply just keep players far enough so they are out of range. Even from just passing by they managed to make me fall for a moment, I can imagine multiple situations in which it would be not only inconvenient but also deadly

![turrets](https://user-images.githubusercontent.com/54917446/114090088-42acf380-98b7-11eb-8801-865644497da5.png)

This one is not really an issue, not sure how to even categorize it. There's just something odd about walls just connecting by corners, there should either be an opening that players can fit into (move that maint room a bit to right or just make it smaller by either one row or column of tiles) or it should be blocked by proper L shaped walls, in a way that telegraphs there is no passage, that its not just some artificial decision by someone who didn't want people to get in there for no real reason and just barely covered the spot

![nopassage](https://user-images.githubusercontent.com/54917446/114090111-493b6b00-98b7-11eb-8311-88833d19b74b.png)

Also weird cam placing outside perma and poster under airlock in maints east from engineering (clicked on it by mistake earlier so its ripped on my screenshot)

![cams](https://user-images.githubusercontent.com/54917446/114090464-c1a22c00-98b7-11eb-9689-4b72da4b2a47.png)

![poster](https://user-images.githubusercontent.com/54917446/114090496-ce268480-98b7-11eb-9335-7e4ac62b83be.png)


From just glancing at it there were some weird empty places in southern maints, not sure if random rooms didn't work there or they are just not filled yet (which is not surprising considering you are still moving things around so final maint shape may change)

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
Security Outposts

Cenrus commented:
![image](https://user-images.githubusercontent.com/31044876/114195036-02498600-9948-11eb-83e4-5e948d454b0e.png)

Spacecurity?

Commit:
Update CorgStation.dmm

Commit:
Fixes turf decals ignoring layers

Commit:
Transport ferry

Commit:
Tram shuttle auto launches

Commit:
Adds lights to the halls

Commit:
Adds lights, cameras and action to the new hallway

Commit:
Update CorgStation.dmm

Commit:
Update CorgStation.dmm

Commit:
Fixes shuttle flight direction, restricts lawyer access

Commit:
Fixes the bridge

Commit:
Moves a misplaced decal

ike709 commented on code/controllers/subsystem/mapping.dm:
```diff
@@ -1,3 +1,5 @@
+//#define FORCE_MAP "_maps/corgstation.json"
+
```
```suggestion
```

ike709 committed:
Update code/controllers/subsystem/mapping.dm

PowerfulBacon commented:
the force maps needs to be there but should be box

ike709 commented:
oops please readd that

PowerfulBacon commented on code/controllers/subsystem/mapping.dm:
```diff
@@ -1,3 +1,5 @@
+//#define FORCE_MAP "_maps/corgstation.json"
```
```suggestion
//#define FORCE_MAP "_maps/boxstation.json"
```

PowerfulBacon committed:
Update mapping.dm
